src/ape/utils.py

```diff
@@ -244,17 +338,34 @@ def __init__(self):
 
     @cached_property
     def ape_org(self):
+        """
+        A property to get organization for the github token.
+        """
         return self._client.get_organization("ApeWorX")
 
     @cached_property
     def available_plugins(self) -> Set[str]:
+        """ "
+        To retrive available plugins to download for the apeworx project.
+
+        Returns:
+            Set[str] (set): datatype to hold plugins.
+        """
         return {
             repo.name.replace("-", "_")
             for repo in self.ape_org.get_repos()
             if repo.name.startswith("ape-")
         }
 
     def get_release(self, repo_path: str, version: str):
+        """
+        Gives version/release number of available plugins.
+
+        Args:
+            repot_path (str): A path to a filesystem
+                                to retrive version/release number.
+            version (str): A string to contain version number.
```

Comment:
```suggestion
            version (str): The version of the release to get.
```

^ You don't have to mention the type twice, like it is already said to be ` (str) ` in the docs, so we good there